packages/worker/src/worker_protocol.ts

```diff
@@ -0,0 +1,333 @@
+/**
+ * Generated by the protoc-gen-ts.  DO NOT EDIT!
```

Comment:
if you're going to check in a generated file, we should at least have a test that asserts it's up-to-date

how hard is it to leave the generated file in bazel-out and not check in as a source file?

Reply:
there are some reasons behind this

- have IDE recognize the file 
- somehow rules_proto does not work on windows. it throws an error with a message "bazel-out is not an external or internal command" which probably has something to do with how rules_proto tries to invoke protoc. so the windows tests are going to fail. I have seen it before.
- generating this file every time is not an option since ts_project won’t pick it up. it might create cycling dependency between worker and typescript package since they are relying on each other to build (not sure though)

I would prefer to add a generated_file_test and make an assertion.  this test would be disabled on windows though because of the problem I have described above.

Reply:
have the assertion test in place: //packages/worker:copy_worker_protocol

Reply:
I couldn’t put the update command comment into the generated file. the assertion fails. so instead I put it into the build file. I hope that's fine